cryptoinvestor/models/user.py

```diff
@@ -1,4 +1,5 @@
 from cryptoinvestor.models.account import Account
+from flask import request
```

Comment:
I don't think models should directly access request data. That is not wise. All the necessary data should be extracted in views and then passed to appropriate methods as arguments.

Reply:
This is unused import now. Please remove it